src/hooks/result/useCategoryFilterList.ts

```diff
@@ -14,79 +16,35 @@ type ReturnType = {
   onCheckItem(name: string): void
 }
 
-export const cateGoryList = [
-  {
-    code: '전체',
-    name: '전체',
-    selected: true
-  },
-  {
-    code: '남성패션/잡화',
-    name: '남성패션/잡화',
-    selected: false
-  },
-  {
-    code: '여성패션/잡화',
-    name: '여성패션/잡화',
-    selected: false
-  },
-  {
-    code: '게임',
-    name: '게임',
-    selected: false
-  },
-  {
-    code: '스포츠/레저',
-    name: '스포츠/레저',
-    selected: false
-  },
-  {
-    code: '장난감/취미',
-    name: '장난감/취미',
-    selected: false
-  },
-  {
-    code: '디지털기기',
-    name: '디지털기기',
-    selected: false
-  },
-  {
-    code: '자동차/공구',
-    name: '자동차/공구',
-    selected: false
-  },
-  {
-    code: '생활가전',
-    name: '생활가전',
-    selected: false
-  },
-  {
-    code: '가구/인테리어',
-    name: '가구/인테리어',
-    selected: false
-  },
-  {
-    code: '도서/티켓/음반',
-    name: '도서/티켓/음반',
-    selected: false
-  },
-  {
-    code: '반려동물',
-    name: '반려동물',
-    selected: false
-  }
-]
-
 const useCategoryFilterList = (): ReturnType => {
-  const [list, setList] = useState<CheckItemType[]>(cateGoryList)
+  const { data: categories, isSuccess } = useGetCategoriesQuery()
+  const [list, setList] = useState<CheckItemType[]>([
+    {
+      code: '',
+      name: '',
+      selected: false
+    }
+  ])
+
   const [selectedCategoryValue, setSelectedCategoryValue] =
-    useState<string>('전체')
+    useState<string>('ALL')
```

Comment:
```suggestion
    useState<string>('ALL')
```
string 대신 카테고리 관련 타입으로 정의되면 좋을 것 같습니다!

Reply:
저도 이부분을 고민했었는데요!!, 
카테고리 리스트 같은 경우 api응답으로 받아와서 서버에서 유동적으로 수정 할 수 있도록 설계 되었습니다...!
그래서 미리타입선언을 할 경우에는 의미가 퇴색되기에 카테고리 응답 type을 string으로 받게 고첬고(main 페이지pr) 여기서도 string으로 받게 일단은 진행 하였습니당

Reply:
@chojooyoung @shinhyojeong 
아하 확인하였습니다! 요거 제가 확인을 놓쳤네요..!
BE에서 타입 변경되었을 때, FE쪽에서 대응하지 않고 string으로 쓰면 어디에서 사이드 이펙트 발생되는지 모를 것 같아서 카테고리 관련 타입으로 정의되는 게 좋을 것 같은데 두분 의견은 어떠신가요?

Reply:
넵 동의합니다~
service 에 타입추가하였습니다!
[링크](https://github.com/price-offer/offer-fe/pull/174/commits/f76538555ccf573ef1003c6c7b3c36e75a841d81#diff-a3ef8cae97136a88c7de463df7fe5d4e73f2c431af0c1f779c5d28b846a5f7ef)